modules/nw-operator-cr.adoc

```diff
@@ -347,6 +347,10 @@ defaultNetwork:
     genevePort: 6081
     ipsecConfig: {}
 ----
+[IMPORTANT]
+====
+Using `OVNKubernetes` can lead to stack exhaustion problem on {ibm-power-name}.
```

Comment:
I suggest changing to "stack exhaustion problems" or "a stack exhaustion problem" as appropriate, for the sake of grammar.

Reply:
Thanks @mramendi 
Updated as: "a stack exhaustion problem".